constructor-manager-ui/src/constructor_manager_ui/style/images.py

```diff
@@ -4,109 +4,9 @@
 #
 # WARNING! All changes made in this file will be lost!
 
-from PyQt5 import QtCore
+from qtpy import QtCore  # type: ignore
```

Comment:
Should this file be ignored from pre-commit and typing validations since it is autogenerated with `pyrcc5` from the `images.qrc` to bundle the images/gif to enable their use?

Maybe adding a section with the steps to update it in case images change (maybe in the [README.md](https://github.com/napari/packaging/blob/main/constructor-manager-ui/README.md)) is needed?

Reply:
>Should this file be ignored from pre-commit and typing validations since it is autogenerated with pyrcc5 from the images.qrc to bundle the images/gif to enable their use?

We could indeed exclude it from pre-commit, however we do need to use qtpy instead of PyQt5, will add script to generate the files and add to README.